### PR TITLE
add qa test of envlibrados for rocksdb

### PR DESCRIPTION
**Pull request by ryneli**
Signed-off-by: Ryne Li <lizhenqiangsnake@gmail.com>

### CODE REVIEW

**Comment by liewegas** [comment]
@jdurgin what do you think?

File: qa/workunits/rados/test_envlibrados_for_rocksdb.sh [hunk 1]
```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```
Comment (jdurgin) [inline qa/workunits/rados/test_envlibrados_for_rocksdb.sh]: looks good, the exit status should indicate if it fails - I like using '#!/bin/bash -ex' for this so we have logging of what precisely failed if something goes wrong

**Comment by jdurgin** [comment]
A couple changes needed: 
1) chmod +x the script 
2) figure out why it's failing 

I pushed it to the wip-rocksdb-librados-test branch and scheduled a run with:

```~/teuthology/virtualenv/bin/teuthology-suite -v -e <my email  addr> -m mira --owner joshd@teuthology -c master -s rados --suite-dir ~/ceph-qa-suite --filter envlibrados --newest 10 --limit 2 workunit_branch.yaml```

where ~/ceph-qa-suite/workunit_branch.yaml contained:
```overrides:
  workunit:
    sha1: 89855ecb68ad4e172978a304b2d8a9566d1c3cf3
```

The results are here:
http://pulpito.ceph.com/joshd-2016-08-22_21:51:53-rados-master---basic-mira/

It failed on ubuntu, I'm guessing from mismatched library/header librados versions, but I'm not sure.
I see some apt-get warnings in the log for the failed run too - these can be fixed by adding ```DEBIAN_FRONTEND=noninteractive``` and ```--force-yes``` to the apt-get commands.

**Comment by jdurgin** [comment]
The failure seems to be a race condition in libnss, or a very subtle bug in common/ceph_crypto.cc. It's not reproducing very reliably anymore, and disappears with timing changes like running manually instead of via scheduled jobs, so it seems safe to merge this. Merged manually to also save the rocksdb test binary for core file analysis.

**Pull request closed without merging**